src/common/event_loop.cc

```diff
@@ -30,11 +30,13 @@ void event_loop_add_file(event_loop *loop,
   /* If it cannot be added, increase the size of the event loop. */
   if (err == AE_ERR && errno == ERANGE) {
     err = aeResizeSetSize(loop, 3 * aeGetSetSize(loop) / 2);
-    CHECK(err == AE_OK);
```

Comment:
I see, this fails if the file descriptor is already in the event loop? And when does that happen? When clients disconnect and then new client reconnect?